src/buildstream/_scheduler/job.py

```diff
@@ -67,43 +67,57 @@ class JobStatus(FastEnum):
 # process. It has some methods that are not implemented - they are meant for
 # you to implement in a subclass.
```

Comment:
There are no subclasses anymore.

Reply:
There are now that I reverted the second part.